bin/package_build.py

```diff
@@ -134,7 +134,7 @@ def fedora():
 				req = requests.get(link)
 				data = req.text
 				if req.status_code == 404:
-					raise Exception(f"404 Directory {dirs[each]} not found")
+					continue
 			except Exception as e:
```

Comment:
@Princee215 let's remove this try - except block and have the try block above at [here](https://github.com/openmainframeproject/software-discovery-tool/pull/162/files#diff-6f39b12c55db3a00f6283da33cbe8fdbf0aa23ef14b767cc7ce7dec7a134654eL127)